cypress/integration/5-editing/editing.spec.js

```diff
@@ -0,0 +1,49 @@
+describe('editing properties', () => {
+  beforeEach(() => {
+    cy.visit('http://localhost:1337');
```

Comment:
Can you change this to use the configured url too? @miram13

Reply:
Ya sure